### ISSUE TITLE
CoffeeScript on NPM has moved to "coffeescript" (no hyphen)

### ISSUE THREAD

**Issue by trickeydan** [PeterJCLaw/srcomp-stream#3]
`npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)`

**Comment by PeterJCLaw** [comment]
I would deeply enjoy this not repo being in CoffeeScript at all.

**Comment by trickeydan** [comment]
> I would deeply enjoy this not repo being in CoffeeScript at all.

As would many of us. I suspect it could be done nicely with something like https://github.com/sysid/sse-starlette

**Comment by PeterJCLaw** [comment]
Yeah, async python is probably the way to go here. I'd be tempted to move the functionality into the HTTP API, which would simplify deployment too.
